.editorconfig

```diff
@@ -6,6 +6,41 @@ insert_final_newline = true
 charset = utf-8
 trim_trailing_whitespace=true
 
+[.editorconfig]
+ij_editorconfig_align_group_field_declarations = false
+ij_editorconfig_space_after_colon = false
+ij_editorconfig_space_after_comma = true
+ij_editorconfig_space_before_colon = false
+ij_editorconfig_space_before_comma = false
+ij_editorconfig_spaces_around_assignment_operators = true
+
+[{*.go,*.go2}]
+ij_continuation_indent_size = 4
```

Comment:
Why not using the standard ones ?
```
indent_style = space
indent_size = 4
```

Reply:
This is what's generated by goland when I export my configuration do you think I should remove them and use standard in the wildcard ?

Reply:
OK I just checked and continuation_indent != indent
This is the indent style when you add a newline
```go
asdfasdf("foo" +
    "bar")
^^^
here   
```

Reply:
Ok I see !